modules/microshift-config-yaml.adoc

```diff
@@ -12,7 +12,7 @@ Configure your {product-title} using a YAML file. The values specified in your c
 [id="microshift-yaml-default_{context}"]
 == Default settings
 If you do not create a `config.yaml` file, the default values are used.
-// Q: what creates this default yaml?
+// Q: what creates the default yaml?
```

Comment:
The `config.yaml.default` file is installed with the RPM as an example of what the file should look like. Nothing reads that file, it's just meant for documentation purposes.